docs/userguide.md

```diff
@@ -386,6 +386,37 @@ status:
   - name: cluster2
 ```
 
+## Aggregated status
+This is an optional feature that can be enabled to collect and aggregate
+the status of target type in member clusters. This can be useful for some
+high-level controllers which can rely on such aggregate status without
+watching individual clusters by themselves.
+
+To enable aggregated status for a federated type configure
+the `FederatedTypeConfig` of that type indicating the same as below:
+For eg: if you want to enable aggregated status for services,
```

Comment:
s/eg:/example,/

Given that aggregated status is already possible for services, perhaps choose a different type?